console/console-init/ui/src/components/InputText/InputText.tsx

```diff
@@ -27,6 +34,7 @@ interface IInputTextProps {
   isExpandable?: boolean;
 }
 const InputText: React.FunctionComponent<IInputTextProps> = ({
```

Comment:
i think name of component should be more meaningful. what you think.
my suggestion is InputTextWithCopy. The purpose of creating this new component for copy, for text/input we have PF TextInput.

Reply:
This will may or may not create Text input with copy. and depends on the props.

Reply:
Also there are too many logics based on props